keras_nlp/samplers/beam_sampler.py

```diff
@@ -36,16 +36,16 @@ class BeamSampler(Sampler):
     Args:
         num_beams: int. The number of beams that should be kept at each
             time-step. `num_beams` should be strictly positive.
-        return_all_beams: bool. When set to `True`, the sampler will return the top prompt,
-            all prompts and their respective probabilities score.
+        return_all_beams: bool. When set to `True`, the sampler will return all
+            beams and their respective probabilities score.
```

Comment:
Let's not mix the docstring change with the random sampler change. Because we squash and merge all commits in a single PR, it will be hard to track. Thanks!

Reply:
I reverted commits on other samplers and I will open a new PR with them